src/Image.php

```diff
@@ -284,20 +284,30 @@ public function listOperations()
      * @param array  $stackOperations Stack operations
      * @param string $organization    Optional organization name
      * @param array  $stackOptions    Stack options
+     * @param bool   $overwrite       If an existing stack should be overwritten
      *
      * @return Stack
      */
-    public function createStack($stackName, array $stackOperations, $organization = '', array $stackOptions = [])
+    public function createStack(
+                                $stackName,
```

Comment:
that indention looks awkward. normally it's just 4 spaces and not based on the function name length.